mirgecom/initializers.py

```diff
@@ -370,6 +465,76 @@ def exact_rhs(self, discr, q, t=0.0):
         return flat_obj_array(massrhs, energyrhs, momrhs)
 
 
+class AcousticPulse:
+    r"""
+    Create an N-dimensional Gaussian acoustic pulse.
+
+    The Gaussian pulse is defined by:
+
+    .. math::
+
+        {\rho}E(\vec{r}) = {\rho}E + amplitude * G(\vec{r})\\
+        G(\vec{r}) = \exp^{-(\frac{(\vec{r}-\vec{r_0})}{\sqrt{2}w})^{2}}
+
+    Where :math:`\vec{r}` are the nodal coordinates, and :math:`\vec{r_0}`,
```

Comment:
```suggestion
        G(\vec{r}) = \exp^{-(\frac{(\vec{r}-\vec{r_0})}{\sqrt{2}w})^{2}},

    where :math:`\vec{r}` are the nodal coordinates, and :math:`\vec{r_0}`,
```
The same rules with all mathematical writing apply here. Math statements are sentences. (Here and elsewhere.)